snuba/query/parser/__init__.py

```diff
@@ -393,15 +393,14 @@ def _expand_aliases(query: Union[CompositeQuery[QueryEntity], Query]) -> None:
 
 
 ARRAYJOIN_FUNCTION_MATCH = FunctionCallMatch(StringMatch("arrayJoin"), None)
+ALIAS_PREFIX = "_snuba_"
 
 
 def _mangle_aliases(query: Union[CompositeQuery[QueryEntity], Query]) -> None:
-    alias_prefix = "_snuba_"
```

Comment:
The SnQL parser also uses this alias prefix, so you should use this constant there as well.

Reply:
Done at all places which were using alias in code except tests.